src/main/java/alfio/model/PromoCodeDiscount.java

```diff
@@ -131,4 +131,56 @@ public static String format(PromoCodeDiscount discount, String eventCurrencyCode
         }
         return MonetaryUtil.formatCents(discount.getDiscountAmount(), eventCurrencyCode);
     }
+
+    public int getId() {
```

Comment:
no need for all the getters, we are using Lombok for auto generating them. You can remove them.